evoting/lib/transaction.go

```diff
@@ -198,6 +201,21 @@ func (t *Transaction) Verify(genesis skipchain.SkipBlockID, s *skipchain.Service
 			return errors.New("ballot user-id differs from transaction user-id")
 		}
 
+		election, err := GetElection(s, genesis, false, t.User)
+		if err != nil {
+			return err
```

Comment:
Please encapsulate errors with `fmt.Errorf` and give some more information what went wrong.